### PR TITLE
added graph properties form, and added a title to the graph

### PR DESCRIPTION
**Pull request by MaderNoob**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,11 +3,19 @@
...
 <body>
     <form>
         <input name="file" accept=".csv" type="file"/>
     </form>
+    <form id="graphPropertiesForm">
+        <label>Line Color: </label><input type="color" name="lineColor" value="#ff0000"/>
+        <label>Graph Title: </label><input type="text" name="graphTitle" value="Graph">
+        <label>X axis title: </label><input type="text" name="xTitle" value="X axis">
+        <label>Y axis title: </label><input type="text" name="yTitle" value="Y axis">
+    </form>
```
Comment (nickBes) [inline index.html:17]: Axis titles are not displayed...

**Review by nickBes — CHANGES_REQUESTED** [review]
meh

File: index.html (hunk 1 above)
Comment (MaderNoob) [reply index.html:17]: No need to draw them now, they will be drawn inside of the graph once the graph movement pull request will be merged

**Review by nickBes — APPROVED** [review]
good

**Pull request merged**